lib/include/DOtherSide/DOtherSideTypes.h

```diff
@@ -61,6 +61,21 @@ typedef void DosQMetaObject;
 /// A pointer to a QObject
 typedef void DosQObject;
 
+/// A pointer to a QQuickImageProvider
+typedef void DosQQuickImageProvider;
+
+typedef void DosPixmap;
+
+struct pixmap_request {
+    const char *id;
+    int *width;
+    int *height;
+    int requestedWidth;
+    int requestedHeight;
+};
+
+typedef DosPixmap* (*pixmap_cb)(pixmap_request request);
```

Comment:
If it's not to much work can you remove the pixmap_request struct...in the other callbacks we didn't use struct...just for consistency. 
While fixing this rename the callback to something like PixmapCallback (so no abbrev. and pascal case)